src/repositories/QueueRepository/__tests__/QueueRepository.int.spec.ts

```diff
@@ -208,4 +209,34 @@ describe("Queue Repository tests", () => {
     expect(DateTime.fromJSDate(playerInDb?.queueTime!).toISO()).toEqual(mockBallChaser.queueTime.toISO());
     expect(playerInDb?.team).toBeNull();
   });
+  it("check if player is in queue", async () => {
```

Comment:
What about when the player is not in the queue? Do we get the correct result?

Reply:
Should I add a negative test, or can I just add a second function call inside of this one and check it for a false.

Reply:
A separate test since it's two different scenarios.

Reply:
fixed in next commit